Open Judge System/Web/OJS.Web/App_GlobalResources/Areas/Administration/ProblemGroups/ProblemGroupsControllers.resx

```diff
@@ -123,8 +123,11 @@
   <data name="Active_contest_cannot_delete_problem_group" xml:space="preserve">
     <value>The contest is аctive and you cannot delete problem groups</value>
   </data>
-  <data name="Cannot_crud_non_online_contest" xml:space="preserve">
-    <value>Cannot manipulate problem groups in contest that is NOT Online</value>
+  <data name="Cannot_create_non_online_contest" xml:space="preserve">
+    <value>You cannot create new problem group from here in NON Online contest.</value>
```

Comment:
Consider changing this to "You cannot create new problem groups in a contest which is not of type {0}" and inserting the type as a dynamically provided variable